app/models/tenant.rb

```diff
@@ -33,7 +33,8 @@ class Tenant < ActiveRecord::Base
   validates :subdomain, :uniqueness => true, :allow_nil => true
   validates :domain,    :uniqueness => true, :allow_nil => true
   validate  :validate_only_one_root
-  validates :name, :description, :presence => true, :unless => :root?
+  validates :description, :presence => true
+  validates :name, :presence => true, :unless => :use_settings?
```

Comment:
Something about `use_settings?` feels weird.  Since it's stored in configurations table (aka VMDB::Config), should be name it `use_configuration?` or `use_config?`? Maybe perhaps even be more explicit with something like `use_config_for_attributes?`?  What do you think?